TRACK_README.md

```diff
@@ -16,57 +16,52 @@ Please read about how to [get involved in a track](https://github.com/exercism/d
 
-- Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
+Before submitting your pull request, you'll want to verify the changes in two ways:
 
-- All the tests for {{LANGUAGE}} exercises can be run from the top level of the repo with ... Please run this command before submitting your PR.
+* Run all the tests for the {{LANGUAGE}} exercises
+* Run an Exercism-specific linter to verify the track
 
-## Contributing a New Exercise ##
+All the tests for {{LANGUAGE}} exercises can be run from the top level of the repo with
 
-- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
+```
+# add this command
+```
 
-- Please make sure the new exercise conforms to specifications in the [exercism/problem-specifications](https://github.com/exercism/problem-specifications) repo.
+For the Exercism-specific linting, please see [the documentation](https://github.com/exercism/docs/blob/master/language-tracks/configuration/linting.md).
 
-- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+## Contributing a New Exercise
 
-- Exercises should use only the {{LANGUAGE}} core libraries.
+Please see the documentation about [adding new exercises](https://github.com/exercism/docs/blob/master/you-can-help/make-up-new-exercises.md).
 
-- Please do not add a README or README.md file to the exercise directory. The READMEs are constructed using shared metadata, which lives in the
-[exercism/problem-specifications](https://github.com/exercism/problem-specifications) repository. Further explanation can be found in [fixing-exercise-readmes](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/exercise-readmes.md)
+Note that:
 
+- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+- Exercises should use only the {{LANGUAGE}} core libraries.
+- Exercises must conform to the Exercism-wide standards described in [the documentation](https://github.com/exercism/docs/tree/master/language-tracks/exercises).
 - Each exercise should have a test suite, an example solution, a template file for the real implementation and ... (anything else that needs to go with each exercise for this track). The CI build expects files to be named using the following convention: (describe the {{LANGUAGE}} convention for naming the various files that make up an exercise).
-
 - Please do not commit any configuration files or directories inside the exercise other than ...
-
-- Be sure to add it to the appropriate place in the `config.json` file. Also, please run `bin/fetch-configlet && bin/configlet` to ensure the exercise is configured correctly.
```

Comment:
good move, because otherwise the two things done by Travis CI are split up in a weird place